dtfs-central-api/api-tests/v1/utilisation-reports/post-payment.api-test.ts

```diff
@@ -267,6 +267,38 @@ describe(`POST ${BASE_URL}`, () => {
     expect(response.status).toEqual(HttpStatusCode.BadRequest);
   });
 
+  it(`should respond with a ${HttpStatusCode.BadRequest} when the selected fee record ids do not match those in the existing fee record payment group`, async () => {
+    // Arrange
+    await SqlDbHelper.deleteAllEntries('Payment');
+    await SqlDbHelper.deleteAllEntries('FeeRecord');
```

Comment:
Should these two be pulled out (and everywhere in this file) to the `beforeEach` or `afterEach`? We seem to be tidying up from other unit tests in the next unit test?

Reply:
I'm a bit disconnected from this -- happy to go with whatever.

If `Payment` and `FeeRecord` tables are populated as part of a `beforeEach` and are not needed as part of this test, I think it's fine to delete these here. But if it is populated in a `beforeEach`, we should also clear in an `afterEach` to ensure our tests don't leak maybe? (which it might be so very sorry if thats the case!)
 
If its clearing up after a previous test, I would wonder if theres a better way of doing this, as if it is cleaning up after an old test, you'll get odd behavior trying to debug the previous test with an `it.only` as itll still have stuff in the db